src/StdToml.sol

```diff
@@ -0,0 +1,177 @@
+// To parse:
+// ```
+// using stdToml for string;
+// string memory toml = vm.readFile("some_path");
+// toml.parseUint("<json_path>");
+// ```
+// To write:
+// ```
+// using stdToml for string;
+// string memory toml = "[key]\n a = 1\n b = 2";
+// toml.serialize("key", "c", 3);
+// toml.write("<some_path>");
+// ```
+
+library stdToml {
+    VmSafe private constant vm = VmSafe(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    function parseRaw(string memory toml, string memory key) internal pure returns (bytes memory) {
+        return vm.parseToml(toml, key);
+    }
+
+    function readUint(string memory toml, string memory key) internal pure returns (uint256) {
+        return vm.parseTomlUint(toml, key);
+    }
+
+    function readUintArray(string memory toml, string memory key) internal pure returns (uint256[] memory) {
+        return vm.parseTomlUintArray(toml, key);
+    }
+
+    function readInt(string memory toml, string memory key) internal pure returns (int256) {
+        return vm.parseTomlInt(toml, key);
+    }
+
+    function readIntArray(string memory toml, string memory key) internal pure returns (int256[] memory) {
+        return vm.parseTomlIntArray(toml, key);
+    }
+
+    function readBytes32(string memory toml, string memory key) internal pure returns (bytes32) {
+        return vm.parseTomlBytes32(toml, key);
+    }
+
+    function readBytes32Array(string memory toml, string memory key) internal pure returns (bytes32[] memory) {
+        return vm.parseTomlBytes32Array(toml, key);
+    }
+
+    function readString(string memory toml, string memory key) internal pure returns (string memory) {
+        return vm.parseTomlString(toml, key);
+    }
+
+    function readStringArray(string memory toml, string memory key) internal pure returns (string[] memory) {
+        return vm.parseTomlStringArray(toml, key);
+    }
+
+    function readAddress(string memory toml, string memory key) internal pure returns (address) {
+        return vm.parseTomlAddress(toml, key);
+    }
+
+    function readAddressArray(string memory toml, string memory key) internal pure returns (address[] memory) {
+        return vm.parseTomlAddressArray(toml, key);
+    }
+
+    function readBool(string memory toml, string memory key) internal pure returns (bool) {
+        return vm.parseTomlBool(toml, key);
+    }
+
+    function readBoolArray(string memory toml, string memory key) internal pure returns (bool[] memory) {
+        return vm.parseTomlBoolArray(toml, key);
+    }
+
+    function readBytes(string memory toml, string memory key) internal pure returns (bytes memory) {
+        return vm.parseTomlBytes(toml, key);
+    }
+
+    function readBytesArray(string memory toml, string memory key) internal pure returns (bytes[] memory) {
+        return vm.parseTomlBytesArray(toml, key);
+    }
+
+    function serialize(string memory jsonKey, string memory rootObject) internal returns (string memory) {
```

Comment:
Looks like a few `json` references slipped by, can you rename the rest of them?
```suggestion
    function serialize(string memory to'lKey, string memory rootObject) internal returns (string memory) {
```

Reply:
Hm, but the example in the comments at the top show this operating on TOML:
```solidity
using stdToml for string;
string memory toml = "[key]\n a = 1\n b = 2";
toml.serialize("key", "c", 3);
toml.write("<some_path>");
```

Reply:
Hi @mds1 I've updated the example with the correct code. This code was an initial draft and incorrect.